### ISSUE TITLE
Adjusting syntax of cheb.x, cheb.xy, and cheb.xyz

### ISSUE THREAD

**Issue by trefethen** [chebfun/chebfun#1816]
The commands `cheb.x`, `cheb.xy`, and `cheb.xyz` are intended to give us quick ways to generate the variable x in 1D, the variables x and y and 2D, and the variables x, y, and z in 3D.  `cheb.x` has been around a while though I don't think we've publicized it.  `cheb.xy` is more important and `cheb.xyz` is much more important -- they add tremendously to convenience of Chebfun/2/3 use.

They're not as convenient as they should be, however, and I would like them to be changed.  Currently
you have to type
```
x = cheb.x ,   [x,y] = cheb.xy ,   [x,y,z] = cheb.xyz
```
In 2D and especially 3D this is annoying.  I think we should revise the codes so that you just type
```
cheb.x ,  cheb.xy ,  cheb.xyz
```
and still they produce results x, y, and z.  The aim of these tools isn't great generality; it's great convenience!

Amusingly, the help texts for `cheb.xy` and `cheb.xyz` already describe what I want these codes to do rather than what they actually do.  For example:
```
>> help cheb.xyz
 xyz   Three chebfun3 obejcts of the identity on [-1, 1, -1, 1, -1, 1].
    cheb.xyz is shorthand for the expressions 
    X = CHEBFUN3(@(X,Y,Z) X),
    Y = CHEBFUN3(@(X,Y,Z) Y), and
    Z = CHEBFUN3(@(X,Y,Z) Z).
```
So if `cheb.xy` and `cheb.xyz` are modified as I request, their help text will require no modification.  Only the help text of `cheb.x` will need to be modified, since it correctly describes what the code currently does:
```
 x   A chebfun of the identity on [-1,1].
    cheb.x is shorthand for the expression CHEBFUN(@(x) x).
```

**Comment by bhashemi** [comment]
I just implemented this feature in chebfun3. A separate pull-request is needed for 2D if we want it to be available before the release of Chebfun3.

**Comment by nickhale** [comment]
I support this proposal. (In fact, for years I've been using an unofficial `chebx()` method with this behaviour on my local copy. Now I can finally go legit by simply adding a `.`.)

**Comment by gradywright** [comment]
Closed by #1859.
